.github/actions/compute-sdk-test/main.js

```diff
@@ -80,19 +117,25 @@ const mainAsyncTask = async () => {
 
   // Iterate through the module tests, and run the Viceroy tests
   for (const moduleKey of moduleKeys) {
+    const testBase = `${fixtureBase}/${moduleKey}`;
+
+    // created/used by ./integration-tests/js-compute/build-one.sh
+    const fastlyTomlPath = `${testBase}/fastly.toml`;
+    const wasmPath = `${testBase}/${moduleKey}.wasm`;
+    const pkgPath = `${testBase}/pkg/${moduleKey}.tar.gz`;
```

Comment:
`pkgPath` appears to be unused; I assume it was only for running tests live in C@E. So I think it should be removed here until we need it.